roosevelt.js

```diff
@@ -362,9 +395,23 @@ module.exports = function (params) {
     } else {
       if (!app.get('params').https.force) {
         serverPush(httpServer, app.get('params').port, 'HTTP')
+        if (httpReload && params.reload.enabled === true) {
+          httpReload.startWebSocketServer().then(function () {
+            logger.log('🎧', `Reload HTTP server is listening on port: ${params.reload.port}`.bold)
+          }).catch(function (err) {
+            logger.error(('Reload was unable to start due to an error' + err.toString()).red)
+          })
+        }
       }
       if (httpsParams.enable) {
         serverPush(httpsServer, httpsParams.port, 'HTTPS')
+        if (httpsReload && params.reload.enabled === true) {
+          httpsReload.startWebSocketServer().then(function () {
+            logger.log('🎧', `Reload HTTPS server is listening on port: ${params.reload.port}`.bold)
+          }).catch(function (err) {
+            logger.error(('Reload was unable to start due to an error' + err.toString()).red)
```

Comment:
I wanted to do `logger.error(('Reload was unable to start due to an error', err).red)` with string interpolation but that didn't seem to work

Reply:
Also I didn't add an emoji, you can tell me what one you want me to do.

Reply:
IIRC the logger adds the default emoji for errors, yes?

Reply:
Not sure if it does in the logger in roosevelt right now does, but the extracted roosevelt-logger module does

Reply:
So something about combining strings, error objects, and colors in a single log messes with the expansion of the object, so in general across the codebase we just drop in an additional `logger.error(err)` so that the user can see the full stack trace.

Reply:
Thanks @Autre31415. Will leave final logging to @kethinov